src/Tests/ConfigUpdaterTest.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+namespace Drupal\commerce\Tests;
+
+use Drupal\simpletest\WebTestBase;
```

Comment:
We're testing no UIs, so it would be much nicer to use a kernel test for this. Can be a followup.